frontend/Filamento.py

```
@@ -53,42 +31,80 @@ def index():
     return redirect(url_for('index'))
 
 
+@app.route('/new', methods=['GET', 'POST'])
+def new():
+    if 'user_logged_in' not in session or session['user_logged_in'] is None:
+        return redirect(url_for('login', following=url_for('new')))
+    return render_template('cadastro.html', title='Filament registration')
+
+
+@app.route('/delete/<int:id>', methods=['GET', 'POST'])
+def delete(id=None):
```

Pylint (3.3.7)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)